MsWheaPkg/HwErrBert/HwErrBert.inf

```diff
@@ -1,70 +1,71 @@
-  BaseMemoryLib
-  CheckHwErrRecHeaderLib
-
-[Protocols]
-  gEfiAcpiTableProtocolGuid           ## CONSUMES
-
-[Pcd]
-  gMsWheaPkgTokenSpaceGuid.PcdVariableHardwareErrorRecordAttributeSupported
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemId
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemTableId
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemRevision
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultCreatorId
-  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultCreatorRevision
-  gMsWheaPkgTokenSpaceGuid.PcdBertEntriesVariableNames
-
-[Guids]
-  gEfiHardwareErrorVariableGuid       ## CONSUMES
-  gMsWheaReportServiceGuid            ## SOMETIMES_CONSUMES
-  gEfiFirmwareErrorSectionGuid        ## SOMETIMES_CONSUMES
-  gEfiEventNotificationTypeBootGuid   ## SOMETIMES_CONSUMES
-  gEfiEventReadyToBootGuid            ## CONSUMES             ## Event
-  gEfiEventExitBootServicesGuid       ## CONSUMES
-
-[Depex]
+## @file -- HwErrorBert.inf
+#
+#  Driver that looks through flash for variables with gEfiHardwareErrorVariableGuid
+#  and publishes them to BERT ACPI table.
+#
+# Copyright (C) Microsoft Corporation. All rights reserved.
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+##
+
+[Defines]
+  INF_VERSION                    = 0x00010005
+  BASE_NAME                      = HwErrorBert
+  FILE_GUID                      = 328AC7D1-6D2B-4F6B-8A2D-EA6302A62F60
+  MODULE_TYPE                    = DXE_DRIVER
+  VERSION_STRING                 = 1.0
+  ENTRY_POINT                    = HwErrorBertEntry
+
+#
+# The following information is for reference only and not required by the
+# build tools.
+#
+#  VALID_ARCHITECTURES           = AARCH64 X64
+#
+
+[Sources]
+  HwErrorBert.c
+  BertHelper.c
+  BertHelper.h
+
+[Packages]
+  MdePkg/MdePkg.dec
+  MdeModulePkg/MdeModulePkg.dec
+  MsWheaPkg/MsWheaPkg.dec
+
+[LibraryClasses]
+  BaseLib
+  UefiLib
+  DebugLib
+  HobLib
+  MemoryAllocationLib
+  PcdLib
+  PrintLib
+  UefiBootServicesTableLib
+  UefiDriverEntryPoint
+  UefiRuntimeServicesTableLib
+  BaseMemoryLib
+  CheckHwErrRecHeaderLib
+
+[Protocols]
+  gEfiAcpiTableProtocolGuid           ## CONSUMES
+
+[Pcd]
+  gMsWheaPkgTokenSpaceGuid.PcdVariableHardwareErrorRecordAttributeSupported
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemId
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemTableId
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultOemRevision
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultCreatorId
+  gEfiMdeModulePkgTokenSpaceGuid.PcdAcpiDefaultCreatorRevision
+  gMsWheaPkgTokenSpaceGuid.PcdBertEntriesVariableNames
+  gMsWheaPkgTokenSpaceGuid.PcdBertEntriesVariableGuid
+
+[Guids]
+  gEfiHardwareErrorVariableGuid       ## CONSUMES
+  gMsWheaReportServiceGuid            ## SOMETIMES_CONSUMES
+  gEfiFirmwareErrorSectionGuid        ## SOMETIMES_CONSUMES
+  gEfiEventNotificationTypeBootGuid   ## SOMETIMES_CONSUMES
+  gEfiEventReadyToBootGuid            ## CONSUMES             ## Event
+  gEfiEventExitBootServicesGuid       ## CONSUMES
+
+[Depex]
```

Comment:
While is this whole file changed? Did line endings get updated?

Reply:
I think this is done by the uncrustify tool. The line ending seems to be the change.

Reply:
Just as long as it is taking the line endings we want :)

Reply:
Yeah, I will try to remove that and run the build once more. Want to avoid that overhaul if possible.

Reply:
Nice catch, that change was not needed and seems to be in the wrong direction. Fixed in the latest update.